source/NVDAObjects/UIA/__init__.py

```diff
@@ -1832,6 +1832,25 @@ def event_UIA_notification(self, notificationKind=None, notificationProcessing=U
 				speech.cancelSpeech()
 			ui.message(displayString)
 
+	def event_UIA_dragComplete(self):
+		"""
+		Raised by an element when a drag operation on it is completed.
+		A prominent example is rearranging Start screen tiles on Windows 8 and later.
+		"""
+		# #12271: speech does not report new location but braille does.

```

Comment:
i do not understand this comment.

Reply:
When I first implemented it (back in November 2019 under Windows 10 App Essentials add-on), I noticed that speech didn't report the newly dragged element but braille did. We may need to find out why and come up with replacements.